default.nix

```diff
@@ -1,13 +1,14 @@
 { pkgs ? import <nixpkgs> { } }:
 
 import ./lib {
-  inherit (pkgs) lib makeScopeWithSplicing';
-  otherSplices = {
-    selfBuildBuild = pkgs.pkgsBuildBuild;
-    selfBuildHost = pkgs.pkgsBuildHost;
-    selfBuildTarget = pkgs.pkgsBuildTarget;
-    selfHostHost = pkgs.pkgsHostHost;
-    selfHostTarget = pkgs.pkgsHostTarget;
-    selfTargetTarget = pkgs.pkgsTargetTarget;
-  };
+  inherit (pkgs)
+    lib
+    makeScopeWithSplicing'
+    splicePackages
+    pkgsBuildBuild
+    pkgsBuildHost
+    pkgsBuildTarget
+    pkgsHostHost
+    pkgsHostTarget
+    pkgsTargetTarget;
```

Comment:
Any reason why this couldn't just be `pkgs.callPackage ./lib {}`?

Reply:
No reason, totally fine to use `pkgs.callPacakge` here!